src/ert/ensemble_evaluator/sync_ws_duplexer.py

```diff
@@ -1,155 +0,0 @@
-from __future__ import annotations
-
-import asyncio
-import contextlib
-import ssl
-import time
-from concurrent.futures import CancelledError
-from typing import AsyncIterable, Iterable, Iterator, Optional, Union
-
-import websockets
-from cloudevents.http import CloudEvent
-from websockets.client import WebSocketClientProtocol
-from websockets.datastructures import Headers
-from websockets.typing import Data
-
-from _ert.async_utils import new_event_loop
-from _ert.threading import ErtThread
-
-from ._wait_for_evaluator import wait_for_evaluator
-
-
-class SyncWebsocketDuplexer:
```

Comment:
Why don't we need this anymore? What part of AsyncMonitor replaced it?

Reply:
One of the reasons to implement async monitor is to get rid of the duplexer (the entity running the async websockets), which now integral part of the monitor.